e2e/extensions-protocol.e2e.spec.ts

```diff
@@ -80,6 +80,8 @@ test.describe.serial('Extensions protocol handler', () => {
     // `buildctl debug info` talks to the backend (to fetch info about it), so
     // if it succeeds it means the backend is up and can respond to requests.
     await retry(() => spawnFile(rdctl, ['shell', 'buildctl', 'debug', 'info']));
+    // On Windows nerdctl needs a bit of time for the containerd socket to be ready
+    await new Promise(resolve => setTimeout(resolve, 1_000));
```

Comment:
I don't understand why you add the delay here instead of in `build and install testing extension` directly before running `nerdctl build`?

But waiting until `nerdctl info` succeeds, as suggested by @mook-as, seems like a better approach.